### PR TITLE
added ethics reporting info; plus WISE contact info

### PR DESCRIPTION
**Pull request by guidodarezzo**

### CODE REVIEW

File: lab_manual.tex [hunk 1]
```diff
@@ -1471,6 +1471,17 @@ \subsection{Interpersonal issues}\label{sec:interpersonal}
 \item Dartmouth's \href{https://sexual-respect.dartmouth.edu/}{Title IX office} (for concerns about sexual misconduct, harassment, or assault)
 
 \item \href{https://www.hanovernh.org/hanover-police-department}{Hanover Police} (if criminal activity is involved)
+
+\item \textbf{WISE}, a 24-hour crisis hotline for free and confidential services to address domestic and sexual violence and stalking. 866 348 9473.
```
Comment (jeremymanning) [inline lab_manual.tex]: minor edit: can you hyphenate the phone number and put it in parentheses?  Also unbold "WISE"

File: lab_manual.tex [hunk 2]
```diff
@@ -1471,6 +1471,17 @@ \subsection{Interpersonal issues}\label{sec:interpersonal}
...
+
+\end{itemize}
+
+\subsection{Ethics issues and reporting}
+\begin{itemize}
+
+\item Dartmouth's \href{https://www.dartmouth.ethicspoint.com}{Compliance and Ethics Hotline}. 888 497 0516.
```
Comment (jeremymanning) [inline lab_manual.tex]: can you hyphenate the phone number and put it in parentheses? (and remove the period after Ethics Hotline)

**Comment by paxtonfitzpatrick** [comment]
hey @guidodarezzo, I've merged a few more recent PRs in the meantime, so just make sure to pull from the upstream repo before making those minor changes. Thanks!

File: lab_manual.tex (hunk 1 above)
Comment (paxtonfitzpatrick) [reply lab_manual.tex]: resolved

File: lab_manual.tex (hunk 2 above)
Comment (paxtonfitzpatrick) [reply lab_manual.tex]: resolved

**Comment by paxtonfitzpatrick** [comment]
implemented requested changes and resolved conflicts -- merging!

**Pull request merged**